joss/paper.md

```diff
@@ -0,0 +1,127 @@
+Because plasma dynamics in the corona are largely confined to the direction of the magnetic field, the explicit dependence on the magnetic field can be neglected and the relevant hydrodynamic equations can be reduced to a single-dimension in space, the coordinate along the coronal loop.
+However, the large range of spatial and temporal scales necessary to resolve this system still make these models computationally expensive enough that large parameter space explorations are prohibitive.
+The enthalpy-based thermal evolution of loops (EBTEL) model [@klimchuk_highly_2008;@cargill_enthalpy-based_2012] mitigates this difficulty by computing spatial integrals of the aforementioned field-aligned hydrodynamic equations.
+Because of its relative simplicity and computational efficiency, EBTEL has been widely used since its initial development [e.g. @qiu_heating_2012;@raftery_multi-wavelength_2009;@ugarte-urra_determining_2014].
+Comparisons to spatially-averaged results from field-aligned hydrodynamic models show very good agreement [@cargill_enthalpy-based_2012-1].
+
+# Statement of Need
+
+EBTEL consists of a set of ordinary differential equations that model the time-dependent behavior of the relevant thermodynamic quantities spatially-averaged over a coronal loop.
+To approximate mass transport within a coronal loop, EBTEL equates an enthalpy flux with a balance between the heat flux out of the corona and the energy lost due to radiation in the transition region (TR), the thin layer of the solar atmosphere just below the corona.
+If TR radiation cannot balance the downward heat flux, this drives an upflow of material into the corona and if the TR is radiating away more energy than the coronal heat flux can supply this drives a downflow.
+This approximation is valid for short loops and bulk velocities below the local sound speed [@klimchuk_highly_2008].
+
+The original software implementation of the EBTEL model of @klimchuk_highly_2008 was in the proprietary Interactive Data Language (IDL).
+Subsequent improvements to the gravitational stratification and radiative losses by @cargill_enthalpy-based_2012 gave better agreement with field-aligned hydrodynamic models[^ebtel2].
+@barnes_inference_2016 modified the EBTEL model to relax the single-fluid assumption and treat electrons and ions separately, allowing for differential heating between the two species and implemented these modifications in C++.
+@cargill_static_2021 later extended EBTEL to include effects due to cross-sectional area expansion and @reep_modeling_2024 added the ability to vary the abundance model for the radiative losses as a function of time.
+
+`ebtelplusplus` unifies all of the aforementioned features into a single set of equations and C++ and Python software implementation.
+In particular, `ebtelplusplus` solves the following equations for the spatially-averaged electron pressure ($p_e$), ion pressure ($p_i$), and density ($n$) of a semi-circular coronal loop of half-length $L$,
```

Comment:
Do any of the references explain why $n_e = n_i$ while the temperatures can differ?  This makes sense in general, but might not be obvious to everyone who isn't terribly familiar with the corona.

Reply:
The appendix of Barnes et al (2016) has a half sentence that states "n_e=n_i=n" due to quasi-neutrality". Maybe one of Steve's papers addresses this more fully.